src/components/Dashboard/Wrappers/gridstack/init-gridstack.ts

```diff
@@ -30,7 +30,7 @@ export const initializeGridstack = (
   newGrid.current = GridStack.init(
     {
       column: columnCount,
-      margin: 10,
+      margin: areaType === 'sidebar' ? 5 : 10,
```

Comment:
Have you tested, that this causes no issues?

Reply:
I have. I don't see any new issues popping up